metrics_proto_test.go

```diff
@@ -30,56 +30,10 @@ import (
 
 	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
 	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
-	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
 	"github.com/golang/protobuf/ptypes/wrappers"
 	"github.com/google/go-cmp/cmp"
 )
 
-func TestProtoResourceToMonitoringResource(t *testing.T) {
```

Comment:
Consider adding a test where proto metrics are exported with different resources, e.g metric1 with GCE resource, then metric2 with K8S resource.

Reply:
done. It uncovered an issue. So thanks for asking to add this test.